src/zope/app/apidoc/codemodule/browser/static_menu.pt

```diff
@@ -7,44 +7,37 @@
     <div>
       <span i18n:translate="">Class Finder:</span>
       <span><i i18n:translate="">(Enter partial Python path)</i></span>
-	</div>
+	  </div>
```

Comment:
The indentation here doesn't match the opening `</div>`.

I get the feeling your editor tried to be helpful, but made things worse?

Reply:
Yeah, I uh, managed to really mess up a lot of things at once here.